go.mod

```diff
@@ -0,0 +1,5 @@
+module github.com/kong/go-echo
+
+go 1.18
+
+require github.com/nu7hatch/gouuid v0.0.0-20131221200532-179d4d0c4d8d
```

Comment:
I'm a bit sad that we're going to use a package from 2013 but 🤷

Reply:
The Go code was copypasta from the other repo, but I agree that having this package is kinda terrible. I replaced it with the latest release of [google uuid](https://github.com/google/uuid)